.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: android
+os: linux
+dist: trusty
+
+android:
+  components:
+    - build-tools-29.0.2
+    - android-29
+
+script:
+  - echo "Travis branch is $TRAVIS_BRANCH"
+  - echo "Travis branch is in pull request $TRAVIS_PULL_REQUEST"
+  - curl "https://sil-storyproducer-resources.s3.amazonaws.com/dev/dev-keystore.jks" -o "dev-keystore.jks"
+  - curl "https://sil-storyproducer-resources.s3.amazonaws.com/dev/keystore.properties" -o "keystore.properties"
+  - ./gradlew test
```

Comment:
It appears that this yml file copies the master jks and properties files from S3 to the local root I think. Can a github guest read this yml file and run it or mimick what it does? thus compromising our protection?

Reply:
That is just a dev keystore for building in CI.  The upload keystore is in a protect S3 bucket.